.azure-pipelines.yml

```diff
@@ -0,0 +1,60 @@
+# This is the configuration file for Azure Pipelines, used to run tests on
+# macOS. Documentation to help understand this setup:
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema
+
+trigger: none # disable builds for branches
+
+jobs:
+- job: macOS
+
+  pool:
+    vmImage: 'macOS-10.13'
+
+  steps:
+  - checkout: self
+    fetchDepth: 50
+    submodules: false
+
+  - script: |
+      echo "Test jobs:"
+      ./wpt test-jobs | while read job; do
+        echo "$job"
+        echo "##vso[task.setvariable variable=run_$job]true";
+      done
+    displayName: 'List test jobs'
+
+  - script: |
+      sudo easy_install pip
+      sudo pip install -U virtualenv
+    displayName: 'Install Python packages'
+    condition: variables.run_wptrunner_infrastructure
+
+  - script: |
+      sudo cp fonts/Ahem.ttf /Library/Fonts
+    displayName: 'Install Ahem font'
+    condition: variables.run_wptrunner_infrastructure
+
+  - script: |
+      # https://github.com/web-platform-tests/results-collection/blob/master/src/scripts/trust-root-ca.sh
+      sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain tools/certs/cacert.pem
+    displayName: 'Install web-platform.test certificate'
+    condition: variables.run_wptrunner_infrastructure
+
+  - script: |
+      # Pin to STP 67, as SafariDriver isn't working in 68.
```

Comment:
How will we remember to update this?

Reply:
I'll need to file a bug and bother @burg about it.